### PR TITLE
[tests-only][full-ci] add test for downloading resouces by user when auto sync is enabled

### PR DESCRIPTION
**Pull request by nabim777**
## Description

In this PR, fllowing scenario are added related to downloading

```feature
-  user download the files shared with them using sharingNg API
- user download the folder shared with them using sharingNg API
```

## Related Issue

- https://github.com/owncloud/ocis/issues/8662

## How Has This Been Tested?

- test environment:
-ci
- locally
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: tests/acceptance/features/apiSharingNg/sharedWithMe.feature [hunk 1]
```diff
@@ -4691,4 +4691,35 @@ Feature: an user gets the resources shared to them
         }
       }
     }
-    """
\ No newline at end of file
+    """
+
+
```
Comment (grgprarup) [inline tests/acceptance/features/apiSharingNg/sharedWithMe.feature:4696]: Why are these scenarios in `sharedWithMe.feature`?

File: services/userlog/pkg/service/l10n/locale/ru/LC_MESSAGES/userlog.po [hunk 2]
```diff
@@ -13,7 +13,7 @@ msgid ""
 msgstr ""
 "Project-Id-Version: \n"
 "Report-Msgid-Bugs-To: EMAIL\n"
-"POT-Creation-Date: 2024-02-29 00:53+0000\n"
+"POT-Creation-Date: 2024-03-20 03:43+0000\n"
```
Comment (saw-jan) [inline services/userlog/pkg/service/l10n/locale/ru/LC_MESSAGES/userlog.po:16]: Please, revert the changes

**Review by saw-jan — CHANGES_REQUESTED** [review]

File: tests/acceptance/features/apiArchiver/downloadById.feature [hunk 3]
```diff
@@ -124,3 +124,40 @@ Feature: download multiple resources bundled into an archive
       | user-agent | archive-type |
       | Linux      | tar          |
       | Windows NT | zip          |
+
+
+  Scenario Outline: user download the folder shared with them using sharingNg API
```
Comment (S-Panta) [inline tests/acceptance/features/apiArchiver/downloadById.feature:129]:
```suggestion
  Scenario Outline: download the folder shared with them
```
it sounds like the user is downloading the file using sharingNG API. Also, isn't this scenario already in the previous step? 
IMO, the given step in the above step should be refactored to sharingNG API. Therefore this scenario might be redundant.

File: tests/acceptance/features/apiArchiver/downloadById.feature [hunk 4]
```diff
@@ -124,3 +124,40 @@ Feature: download multiple resources bundled into an archive
...
+    And the downloaded <archive-type> archive should contain these files:
+      | name                       | content  |
+      | folderToShare/textfile.txt | to share |
+    Examples:
+      | user-agent | archive-type |
+      | Linux      | tar          |
+      | Windows NT | zip          |
+
+
+  Scenario: user download the file shared with them using sharingNG API
```
Comment (S-Panta) [inline tests/acceptance/features/apiArchiver/downloadById.feature:152]:
```suggestion
  Scenario: user download the file shared with them
```

**Comment by S-Panta** [comment]
Also, `auto-sync enabled` is the default condition for ocis . It generally mean downloading the file/folder shared to receiver

**Comment by nabim777** [comment]
The scenario will be covered in this line while doing the issue 8717.
- https://github.com/owncloud/ocis/blob/8f0b536ef75c552703d84e5e541d99727e6a5c31/tests/acceptance/features/apiSpacesDavOperation/getFileByFileId.feature#L11
- https://github.com/owncloud/ocis/blob/8f0b536ef75c552703d84e5e541d99727e6a5c31/tests/acceptance/features/apiArchiver/downloadById.feature#L75

So, this PR is closed since it will be raised as a duplicate scenario.

**Pull request closed without merging**